pkg/cli/jobtemplate/jobtemplate_test.go

```diff
@@ -0,0 +1,417 @@
+package jobtemplate
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/spf13/cobra"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	flowv1alpha1 "volcano.sh/apis/pkg/apis/flow/v1alpha1"
+)
+
+func TestListJobTemplate(t *testing.T) {
+	testCases := []struct {
+		name        string
+		Response    interface{}
+		Namespace   string
+		ExpectedErr error
+	}{
+		{
+			name: "Normal Case",
+			Response: &flowv1alpha1.JobTemplateList{
+				Items: []flowv1alpha1.JobTemplate{
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:      "test-jobtemplate",
+							Namespace: "default",
+						},
+					},
+				},
+			},
+			Namespace:   "default",
+			ExpectedErr: nil,
```

Comment:
It's better to also test the print result, not just err: )

Reply:
done